sympy/functions/combinatorial/tests/test_comb_factorials.py

```diff
@@ -471,6 +471,18 @@ def test_binomial():
     assert expand_func(binomial(n, -2, evaluate=False)) == 0
     assert expand_func(binomial(n, k)) == binomial(n, k)
 
+def test_multinomial():
+    assert multinomial(12, 5, 4, 3) == 27,720
+    assert multinomial(3, 2, 0, 1) == 3
+    assert multinomial(4, 2, 1, 1) == 12
+    assert multinomial(5, 2, 2, 1) == 30
+    assert multinomial(17, 4, 12, 1) == 30940
+    assert multinomial(149, 144, 1, 4) == 2859400145
+    assert multinomial(-5, -2, -2, -1) == 0
+    assert multinomial(-3, -2, 0, -1) == 0
+    assert multinomial(-12, -5, -4, -3) == 0
+    assert multinomial(-4, -2, -1, -1) == 0
```

Comment:
What happens if pass symbols to `multinomial` like, `k1`, `k2`, `k3`?

Reply:
How `multinomial.rewrite` works?

Reply:
Do you want a test case where we use variables?

Reply:
do look at my [tests](https://github.com/sympy/sympy/pull/18960/commits/e953696d24cdeded902fb9e6e4ddba3f7f989b90)

Reply:
> Do you want a test case where we use variables?

Yes, kind of.